vllm/engine/llm_engine.py

```diff
@@ -334,29 +348,82 @@ def _verify_args(self) -> None:
             self.lora_config.verify_with_scheduler_config(
                 self.scheduler_config)
 
+    def _add_request(
+        self,
+        request_id: str,
+        processed_inputs: LLMInputs,
+        sampling_params: SamplingParams,
+        arrival_time: float,
+        lora_request: Optional[LoRARequest],
+    ) -> None:
```

Comment:
What's the motivation of adding this private method and moving the implementation from `add_request` to `_add_request`?

Perhaps we should also call this method `_add_processed_request`?

Reply:
See the above comment re`_AsyncLLMEngine.add_request_async` originally called `add_request` with the processed args. Since the new `add_request` now only accepts the raw args, I have moved the core logic of `add_request` into an inner method `_add_request`, with the outer `add_request` only responsible for processing the raw args.

I'll rename the method as you have suggested to be more clear.